index.html

```diff
@@ -3,8 +3,8 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/dist/output.css">
     <title>Website project template</title>
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
Why did you remove the normal CSS Output file that we are using in this project?
If you wanted to add an external CDN file you could have added it without removing the `output.css` file that we were already using!

```suggestion
    <link rel="stylesheet" href="/dist/output.css">
    <script src="https://cdn.tailwindcss.com"></script>
```